packages/protocol/src/interfaces/ILendingProvider.sol

```diff
@@ -124,4 +124,18 @@ interface ILendingProvider {
    * - Must not require Vault context.
    */
   function getBorrowRateFor(IVault vault) external view returns (uint256 rate);
+
+  /**
+   * @notice Collects rewards from the protocol. Returns false if there are no rewards to be collected.
+   *
+   * @param data bytes to be used to call the harvest function at the lending provider.
+   *
+   */
+  function harvest(bytes memory data) external returns (bool success);
+
+  /**
+   * @notice Returns the address of the `token` that is harvested from the protocol.
+   *
+   */
+  function getHarvestToken() external view returns (address token);
```

Comment:
What about there is more than 1 token to harvest, like in the case of Morpho?

Reply:
good point. I will change to return address[] tokens . I've also added IVault as parameter because I've noticed a lot of them depend on the asset (makes sense).